autogpt_platform/backend/backend/blocks/example/example.py

```diff
@@ -0,0 +1,143 @@
+import logging
+from typing import Any
+
+from backend.data.block import Block, BlockCategory, BlockOutput, BlockSchema, BlockType
+from backend.data.model import (
+    APIKeyCredentials,
+    ContributorDetails,
+    CredentialsField,
+    SchemaField,
+)
+
+from ._api import ExampleClient
+from ._auth import TEST_CREDENTIALS, TEST_CREDENTIALS_INPUT, ExampleCredentialsInput
+
+logger = logging.getLogger(__name__)
+
+
+class ExampleBlock(Block):
+
+    class Input(BlockSchema):
+        name: str = SchemaField(
+            description="The name of the example block", placeholder="Enter a name"
+        )
+        greetings: list[str] = SchemaField(
+            description="The greetings to display", default=["Hello", "Hi", "Hey"]
+        )
+        is_funny: bool = SchemaField(
+            description="Whether the block is funny", placeholder="True", default=True
+        )
+        # Only if the block needs credentials
+        credentials: ExampleCredentialsInput = CredentialsField(
+            description="The credentials for the example block"
+        )
+
+    class Output(BlockSchema):
+        response: dict[str, Any] = SchemaField(
```

Comment:
Since this is a best practice, I think we should avoid using dict and use a proper object return type

Reply:
Like this?

```python
class Output(BlockSchema):
    response: SomePydanticClass = SchemaField(...
```

Reply:
Done